spec/models/waste_exemptions_engine/new_registration_workflow_state/operator_name_form_spec.rb

```diff
@@ -14,29 +14,6 @@ module WasteExemptionsEngine
           expect(new_registration).to transition_from(current_state).to(next_state).on_event(:next)
         end
 
-        context "when neither new_registration.partnership? nor " \
```

Comment:
Has this test logic moved somewhere else?

Reply:
As discussed - this is fine, the specs that were removed are no longer valid and the ones that remain are all that is now needed.